mermin/src/metrics/flow.rs

```diff
@@ -2,6 +2,17 @@
 
 use crate::metrics::registry;
 
+/// Increment the flow events counter.
+///
+/// ### Arguments
+///
+/// - `event_type` - Type of flow event: "received", "dropped_backpressure", or "dropped_error"
+pub fn inc_flow_events(event_type: &str) {
```

Comment:
Let's make `event_type` an enum within this file. For example `FlowEventResult`.